stories/RadioButtonGroup-stories.tsx

```diff
@@ -20,6 +20,7 @@
 
 import type { Meta, StoryObj } from '@storybook/react';
 import { RadioButtonGroup } from '../src';
+import { RadioButtonGroupAlignment } from '../src/components/radio-button-group/RadioButtonGroup';
```

Comment:
RadioButtonGroupAlignment should be re-exported and imported on the same line as the one above